bin/make_release

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# `make_release TAG` creates a Github release for git tag TAG
+
+cd "$(git rev-parse --show-toplevel)"
+
+# the first parameter should be the name of an annotated tag in the git repo
+RELEASE="$1"
+RELEASE_NOTES_FILE=build/release-notes
+IMAGE=fiaas/fiaas-deploy-daemon
+# the following files under build/ are artifacts expected to be created by the initial build stage
+# the container image tarball
+IMAGE_TARBALL=build/fiaas-deploy-daemon.tar
+# contains the container image version as a single line
+IMAGE_VERSION="$(head -n1 build/version)"
+
+function update_chart_release_version() {
+    local release="$1"
+    # strip the `v` prefix of the tag name: the `version` key in Chart.yaml must be valid semver 2.0.0
+    local chart_version="${release##v}"
+
+    for file in Chart.yaml.tpl values.yaml.tpl; do
+        local template_path="helm/fiaas-deploy-daemon/$file"
+        local result_path="helm/fiaas-deploy-daemon/${file%%.tpl}" # remove .tpl suffix
+        echo "Setting RELEASE_VERSION=${release} CHART_VERSION=${chart_version} in $result_path"
+        # '$RELEASE_VERSION' and '$CHART_VERSION' not supposed to be expanded below; it tells envsubst to substitute
+        # those variables only
+        # shellcheck disable=SC2016
+        RELEASE_VERSION="$release" CHART_VERSION="$chart_version" \
+                       envsubst '$RELEASE_VERSION $CHART_VERSION' < "$template_path" > "$result_path"
+    done
+}
+
+function publish_helm_chart() {
+    local release="$1"
+
+    update_chart_release_version "$release"
+    # helm package outputs 'Successfully packaged chart and saved it to: path/to/chart.tgz',
+    # remove 'Successfully packaged ...' prefix to get path to chart tarball
+    local helm_package_output
+    helm_package_output="$(helm package helm/fiaas-deploy-daemon)"
+    local chart_tarball="${helm_package_output##Successfully packaged chart and saved it to: }"
```

Comment:
The reason for parsing the output of the command here is to get the full path to the tarball, including its filename. The `-d` option only makes it possible to set the directory the tarball is created in, so to get the full path from that it would be necessary to guess or infer the filename.